src/main/webapp/scripts/sys/isoFipsCountryMapScript.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * CU Generic ISO/FIPS Country modification
+ */
+const ISO_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.isoCountry.name.div';
+const FIPS_COUNTRY_NAME_FIELD = 'document.newMaintainableObject.fipsCountry.name.div';
+const EMPTY_STRING = '';
+
+function onblur_lookupFipsCountry(fipsCountryCodeField) {
+    let primaryKeyValue = ensurePrimaryKeyIsUppercase(dwr.util.getValue(fipsCountryCodeField));
```

Comment:
Is "fipsCountryCodeField" the field object itself, or just the field's ID? That's what I was trying to explain in the related conversation on this PR. If it's the former but the script is still working fine, then this could be left as-is for now, though we may want to reevaluate the setup at some future point.

The same goes for the similar spot in the  "onblur_lookupIsoCountry" function.

Reply:
Browser console logging showed this for the fipsCountryCodeField and isoCountryCodeField    [object HTMLInputElement]    so it is the object.

I have changed the code back to the previous version so that there is only one point of modification for the "dwr.util.getValue" call if that is needed in the future. From what I can see running locally, document is working as expected.